docs/building.md

```diff
@@ -528,6 +528,44 @@ Executing `Packaging/miyoo_mini/build.sh` will create the folder `build-miyoo-mi
 OnionOS Port Collection.
 </details>
 
+<details><summary>macOS 10.4 Tiger</summary>
+
+For macOS Tiger, DevilutionX can be compiled using the compiler and libraries from [MacPorts](https://www.macports.org/).
+
+For PowerPC, you can use precompiled dependencies from here:
+
+http://macports-tiger-ppc.glebm.com/
+
+After installing MacPorts, run:
+
+~~~ bash
+# Some packages may require you to manually deactivate certain ports during installation.
+# Remember to reactivate them after installing.
+sudo port install curl curl-ca-bundle gcc14 cmake \
+  libsdl12 libsdl_image libsodium libfmt11 bzip2 zlib lua54 \
```

Comment:
libfmt11 isn't being used so no need to build it

Reply:
Yeah, also gtest and google-benchmark aren't used.
I'll removed them if the upstream issues aren't fixed by the time this is ready to merge

Reply:
Done